src/store/modules/job/actions.ts

```diff
@@ -496,47 +496,6 @@ const actions: ActionTree<JobState, RootState> = {
       delete cached[payload.inputFields.systemJobEnumId];
     }
 
-    // added condition to store multiple pending jobs in the state for order batch jobs,
-    // getting job with status Service draft as well, as this information will be needed when scheduling
-    // a new batch
-    // TODO: this needs to be updated when we will be storing the draft and pending jobs separately
-    const batchJobEnums = JSON.parse(process.env?.VUE_APP_BATCH_JOB_ENUMS as string)
-    let batchJobEnumIds = Object.values(batchJobEnums)?.map((job: any) => job.id);
-    // If query is for single systemJobEnumId only update it 
-    if (typeof payload.inputFields.systemJobEnumId === "string" && batchJobEnumIds.includes(payload.inputFields.systemJobEnumId)) {
-      batchJobEnumIds = [ payload.inputFields.systemJobEnumId ];
-    } else if (payload.inputFields.systemJobEnumId && typeof payload.inputFields.systemJobEnumId === "object") {
-      batchJobEnumIds = batchJobEnumIds.filter((batchJobEnumId: any) => payload.inputFields.systemJobEnumId.includes(batchJobEnumId));
-    } else {
-      // If we are not explicitly getting the batch jobs skip updating it in cache
-      batchJobEnumIds = [];
-    }
-    batchJobEnumIds.map((batchBrokeringJobEnum: any) => {
-      cached[batchBrokeringJobEnum] = responseJobs.filter((job: any) => job.systemJobEnumId === batchBrokeringJobEnum).reduce((batchBrokeringJobs: any, job: any) => {
-        if (job.statusId === 'SERVICE_DRAFT') delete job.runTime;
-        batchBrokeringJobs.push({
-          ...job,
-          id: job.jobId,
-          frequency: job.tempExprId,
-          enumId: job.systemJobEnumId,
-          status: job.statusId
-        })
-        return batchBrokeringJobs;
-      }, [])
-    })
-    
-    // added condition to store multiple pending jobs in the state for order batch jobs  
-    responseJobs.filter((job: any) => !batchJobEnumIds.includes(job.systemJobEnumId) && job.statusId === 'SERVICE_PENDING').reduce((cached: any, job: any) => {
```

Comment:
Do not need to remove this logic, as its used to store pending jobs in state and not only dependent on batch jobs